platform-sdk/swirlds-virtualmap/src/main/java/com/swirlds/virtualmap/VirtualMap.java

```diff
@@ -223,6 +223,10 @@ VirtualRootNode<K, V> getRoot() {
         return root;
     }
 
+    public void fullLeafRehash() {
```

Comment:
This looks like a temporary workaround for data migration. Please, file a new ticket to removed it later (along with a few other things like getFirst/LastLeafPath() in VirtualDataSource).

Reply:
Sure, will do.